README.md

```diff
@@ -110,7 +110,6 @@ npm test
 npm run test:headless
 
 # debug or run a subset of local unit tests
-# then navigate to `http://localhost:9876/debug.html`
```

Comment:
The new CLI tooling has a nice interface that tells you to "Press D to debug in a browser" and it'll automatically open the browser and let you drill into individual tests. So this isn't needed.